src/main/java/org/project/specifications/PlaylistSpec.java

```diff
@@ -21,13 +21,18 @@ public RequestSpecification getPlaylistUpdateSpec(String playlistId, Playlist up
                 .setBody(updatedPlaylist)
                 .build();
     }
+    public RequestSpecification getPlaylistGetSpec(String playlistId) {
+        return baseRequestBuilder
```

Comment:
for this request you should use different request builder, without content-type json. Because for get response you do not send any data in request body